app/cells/plugins/core/user_cell.rb

```diff
@@ -14,7 +14,7 @@ def value
       end
 
       def render_select
-        @options[:form].select 'data[user_id]', user_data_for_select, {selected: value}
+        @options[:form].select 'data[user_id]', user_data_for_select, {selected: value}, required: required?
       end
 
       def user_data_for_select
```

Comment:
I only applied it to fields that would trigger the HTML5 validation, which means that only DateTime (text input), Float / Integer (Number Field), Tag (Text Input), Text (Text Input), and User (Dropdown Select) have the ability to have on form HTML5 required validations